src/test/java/co/com/api/test/ApiHttpBinTest.java

```diff
@@ -1,5 +1,9 @@
 package co.com.api.test;
```

Comment:
why these changes, are you updated with main?

Reply:
Because when i configurate the spaces and other checkstyle configuration, with the indention in intellij the spaces are removed and i need to restores this.